server/cmwell-dc/src/pack/logback.xml

```diff
@@ -158,7 +158,7 @@
             <cleanHistoryOnStart>true</cleanHistoryOnStart>
         </rollingPolicy>
         <encoder>
-            <pattern>%date{ISO8601} %-44.44([%thread]) %-5level %-36.36logger{36} - %msg%n</pattern>
+            <pattern>%date{ISO8601} %-44.44([%thread]) %-5level %-72.72logger{72} - %msg%n</pattern>
```

Comment:
changing the logger (module) length will break the log parser. see:
https://github.com/thomsonreuters/CM-Well/blob/f7da7b700e619f42c236f201d281cf100285a933/server/cmwell-cons/app/bin/utils/cm-well-formats.json#L9

The logger part of the was was already abused too much. It should be part of the message.